tests/suites/helpers.function

```diff
@@ -601,7 +601,7 @@ static int uecc_rng_wrapper( uint8_t *dest, unsigned int size )
 }
 #endif /* MBEDTLS_USE_TINYCRYPT */
 
-#if defined(MBEDTLS_ENTROPY_HARDWARE_ALT)
+#if defined(MBEDTLS_ENTROPY_HARDWARE_ALT) && !defined(DEVICE_TRNG)
```

Comment:
Will we run we device TRNG if it is defined ?

Reply:
If it is defined then the hardware poll function is defined here: https://github.com/PelionIoT/pmi-arm-firmware/blob/master/libs/mbed-os/features/mbedtls/platform/src/mbed_trng.cpp